dbms/src/IO/ReadHelpers.h

```diff
@@ -39,7 +39,7 @@
 #endif
 
 
-#define DEFAULT_MAX_STRING_SIZE 0x00FFFFFFULL
+#define DEFAULT_MAX_STRING_SIZE 0x3FFFFFFFULL
```

Comment:
Missing comment here. Why 3F, how it is motivated?

Reply:
The intermediate state of bitmap is also contained within the query to be transferred among nodes, it's very easy to exceed the original 16MB size for a bitmap state and it would be cut leading to parsing exception.

Reply:
bitmap serialization writes each UInt32 as 4 bytes, and become at most 8 bytes when escaped. 0x3FFFFFFFULL can cover at least (2^30)/8=125M UInt32 integers.
We could reduce bitmap serialization size by using the official CRoaring serialization. This also could help to improve bitmap serialization/deserialization performance.

Reply:
Ok.

> This also could help to improve bitmap serialization/deserialization performance.

It will be appreciated if you can implement it with backwards compatibility.